### ISSUE TITLE
Temperature Unit discrepancies between sensors

### ISSUE THREAD

**Issue by Landshark77** [briis/weatherbit#46]
Running 1.0.1.

Noticed the weather.weatherbit sensor aligns with my "Unit System" settings I have in home assistant, yet the daily forecast sensors (sensor.weatherbit_forecast_day_x) default to Celsius.   Is it possible to have all sensors align to the Unit System settings defined in Home Assistant?

weather.weatherbit

```
temperature: 61
humidity: 84
ozone: 259.4
pressure: 30.08
wind_bearing: 168
wind_speed: 6.49
visibility: 10.28
forecast:
  - datetime: '2021-12-23T18:00:00+00:00'
    temperature: 72.1
    templow: 51.8
    precipitation: 0
    precipitation_probability: 0
    condition: partlycloudy
    wind_speed: 6.49
    wind_bearing: 168
  - datetime: '2021-12-24T18:00:00+00:00'
    temperature: 76.1
    templow: 59.4
    precipitation: 0
    precipitation_probability: 0
    condition: partlycloudy
    wind_speed: 8.95
    wind_bearing: 191
```

sensor.weatherbit_forecast_day_1
```

attribution: Powered by Weatherbit.io
datetime: '2021-12-23T18:00:00+00:00'
temperature: 22.3
templow: 11
precipitation: 0
precipitation_probability: 0
snow: 0
cloudiness: 35
weather_text: Scattered clouds
wind_speed: 2.9
wind_bearing: 168
icon: mdi:weather-partly-cloudy
friendly_name: Weatherbit Forecast Day 1
```

sensor.weatherbit_forecast_day_2

```
attribution: Powered by Weatherbit.io
datetime: '2021-12-24T18:00:00+00:00'
temperature: 24.5
templow: 15.2
precipitation: 0
precipitation_probability: 0
snow: 0
cloudiness: 33
weather_text: Scattered clouds
wind_speed: 4
wind_bearing: 191
icon: mdi:weather-partly-cloudy
friendly_name: Weatherbit Forecast Day 2
```

**Comment by briis** [comment]
Oh yes. I forgot to convert them. Will be in the next release.

**Comment by briis** [comment]
Just released 1.0.2 where this is now fixed.
